<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QMainForm</class>
 <widget class="QWidget" name="QMainForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>732</width>
    <height>351</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Detector de Matriculas AOC 2012-13</string>
  </property>
  <widget class="QFrame" name="frameOrig">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>320</width>
     <height>100</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QFrame" name="frameDest">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>40</y>
     <width>320</width>
     <height>100</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonCargar">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>0</y>
     <width>71</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Cargar</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelOrig">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>10</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sans Serif</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Matrícula </string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelDest">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>10</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sans Serif</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Matricula filtrada</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonCopiarO">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>100</y>
     <width>51</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>&gt;&gt;&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonCopiarD">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>50</y>
     <width>51</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;&lt;&lt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonBorrar">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>150</y>
     <width>71</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Borrar</string>
   </property>
  </widget>
  <widget class="QFrame" name="options">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>281</width>
     <height>51</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="pushButtonNegar">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>111</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Invertir Img.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonUmbralizar">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>10</y>
      <width>131</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Umbralizar Img.</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="labelMat">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>230</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sans Serif</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>0000 XXX</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonSalir">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>310</y>
     <width>71</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Salir</string>
   </property>
  </widget>
  <widget class="QFrame" name="options_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>281</width>
     <height>121</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QSpinBox" name="spinUdetv">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>30</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>umbral detección</string>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>20</number>
    </property>
    <property name="value">
     <number>13</number>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonDetectarH">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>60</y>
      <width>141</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Detectar Letras</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonDetectarV">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>30</y>
      <width>141</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Det. Margen Vert.</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelDest_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>31</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sans Serif</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Umb. H.</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelDest_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>62</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sans Serif</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Umb. V.</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinUdeth">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>61</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>umbral detección</string>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>20</number>
    </property>
    <property name="value">
     <number>3</number>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="options_3">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>260</y>
     <width>271</width>
     <height>81</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QSpinBox" name="spinAncho">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>10</y>
      <width>60</width>
      <height>20</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>umbral detección</string>
    </property>
    <property name="minimum">
     <number>20</number>
    </property>
    <property name="maximum">
     <number>50</number>
    </property>
    <property name="value">
     <number>32</number>
    </property>
   </widget>
   <widget class="QLabel" name="labelDest_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sans Serif</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Ancho:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelDest_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sans Serif</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Alto:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinAlto">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>50</y>
      <width>60</width>
      <height>20</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>umbral detección</string>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="value">
     <number>55</number>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonRecortar">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>10</y>
      <width>81</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Recortar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonMatching">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>40</y>
      <width>81</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Matching</string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frameD0">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>160</y>
     <width>32</width>
     <height>55</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QFrame" name="frameD1">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>160</y>
     <width>32</width>
     <height>55</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QFrame" name="frameD2">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>160</y>
     <width>32</width>
     <height>55</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QFrame" name="frameD3">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>160</y>
     <width>32</width>
     <height>55</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QFrame" name="frameD4">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>160</y>
     <width>32</width>
     <height>55</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QFrame" name="frameD5">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>160</y>
     <width>32</width>
     <height>55</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QFrame" name="frameD6">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>160</y>
     <width>32</width>
     <height>55</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonTodo">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>220</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">&lt;font color=\&quot;blue\&quot;&gt;</string>
   </property>
   <property name="text">
    <string>RECONOCER</string>
   </property>
  </widget>
  <zorder>options</zorder>
  <zorder>frameOrig</zorder>
  <zorder>frameDest</zorder>
  <zorder>pushButtonCargar</zorder>
  <zorder>labelOrig</zorder>
  <zorder>labelDest</zorder>
  <zorder>pushButtonCopiarO</zorder>
  <zorder>pushButtonCopiarD</zorder>
  <zorder>pushButtonBorrar</zorder>
  <zorder>labelMat</zorder>
  <zorder>pushButtonSalir</zorder>
  <zorder>options_2</zorder>
  <zorder>options_3</zorder>
  <zorder>frameD0</zorder>
  <zorder>frameD1</zorder>
  <zorder>frameD2</zorder>
  <zorder>frameD3</zorder>
  <zorder>frameD4</zorder>
  <zorder>frameD5</zorder>
  <zorder>frameD6</zorder>
  <zorder>pushButtonTodo</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
